test/Interop/Cxx/foreign-reference/Inputs/module.modulemap

```diff
@@ -58,3 +58,8 @@ module FunctionsAndMethodsReturningFRT {
   header "cxx-functions-and-methods-returning-frt.h"
   requires cplusplus
 }
+
+module CxxFrtInheritance {
```

Comment:
Let's merge this with the existing `module Inheritance`. Same for the `.swift` test files.